test/mockPaymentRequest.js

```diff
@@ -0,0 +1,33 @@
+const paymentRequest = {
+  agreementNumber: 'SIP000000000001',
+  contractNumber: 'SFIP000001',
+  correlationId: '9b232ff2-d0bc-4f0a-993b-3754d66b61b1',
+  currency: 'GBP',
```

Comment:
Should we have consts for things like this, ledger, schedule etc yet?  Or do you think it's premature given the app doesn't actually do anything yet.